File: SEM4-C/ejer2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SEM4_C
{
    internal class ejer2
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese el año: ");
            int an = int.Parse(Console.ReadLine());

            if ((an % 4 == 0 & an %100 != 0) || an%400 == 0)
            {
                Console.WriteLine("\nEl año es bisiesto");
            }else Console.WriteLine("\nEl año no es bisiesto");

            if(an%2 == 0) Console.WriteLine("\nEl año es par");
            else Console.WriteLine("\nEl año es impar");
        }
    }
}

File: SE4-P/ejer2.py
an = int(input("Ingrese el año: "))

if ((an % 4 == 0 and an %100 != 0) or an%400 == 0):
    print("\nEl año es bisiesto")
else: print("\nEl año no es bisiesto")

if(an%2 == 0): print("\nEl año es par");
else: print("\nEl año es impar");